### PR TITLE
[release-v1.33]  Auto Release Update for v1.33.0

### PR DESCRIPTION
**Pull request by danudey**

### CODE REVIEW

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and delete the branch after I've merged it.

File: config/common_versions.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
-title: master
+title: v1.1.18
 components:
   key-cert-provisioner:
     image: tigera/key-cert-provisioner
-    version: master
+    version: v1.1.18
```
Comment (rene-dekker) [inline config/common_versions.yml:5]: I thought we could use v1.1.17 again

Comment (rene-dekker) [reply config/common_versions.yml:5]: Nvm, I see 1.1.18 is already tagged, so we might as well use that.

**Review by rene-dekker — APPROVED** [review]
lgtm

**Comment by marvin-tigera** [comment]
I'm sorry but I failed delete the branch after merging the pull request.

**Pull request merged**